### ISSUE TITLE
`yarn test` fails

### ISSUE THREAD

**Issue by aahutsal** [solana-labs/solana-web3.js#1008]
cloned repository, followed instructions (`rust` installed)
got issue running `yarn test`

here's a full log:
```bash
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/home/gitpod/.nvm/versions/node/v12.20.0/bin/node',
1 verbose cli   '/workspace/solana-web3.js/node_modules/.bin/npm',
1 verbose cli   'run',
1 verbose cli   'build:fixtures'
1 verbose cli ]
2 info using npm@6.14.10
3 info using node@v12.20.0
4 verbose run-script [ 'prebuild:fixtures', 'build:fixtures', 'postbuild:fixtures' ]
5 info lifecycle @solana/web3.js@0.0.0-development~prebuild:fixtures: @solana/web3.js@0.0.0-development
6 info lifecycle @solana/web3.js@0.0.0-development~build:fixtures: @solana/web3.js@0.0.0-development
7 warn lifecycle The node binary used for scripts is /tmp/yarn--1608908653069-0.02024881727038297/node but npm is using /home/gitpod/.nvm/versions/node/v12.20.0/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
8 verbose lifecycle @solana/web3.js@0.0.0-development~build:fixtures: unsafe-perm in lifecycle true
9 verbose lifecycle @solana/web3.js@0.0.0-development~build:fixtures: PATH: /workspace/solana-web3.js/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/workspace/solana-web3.js/node_modules/.bin:/tmp/yarn--1608908653069-0.02024881727038297:/workspace/solana-web3.js/node_modules/.bin:/home/gitpod/.config/yarn/link/node_modules/.bin:/home/gitpod/.yarn/bin:/home/gitpod/.nvm/versions/node/v12.20.0/libexec/lib/node_modules/npm/bin/node-gyp-bin:/home/gitpod/.nvm/versions/node/v12.20.0/lib/node_modules/npm/bin/node-gyp-bin:/home/gitpod/.nvm/versions/node/v12.20.0/bin/node_modules/npm/bin/node-gyp-bin:/home/gitpod/.nvm/versions/node/v12.20.0/bin:/home/gitpod/.pyenv/plugins/pyenv-virtualenv/shims:/home/gitpod/.pyenv/shims:/workspace/.cargo/bin:/workspace/.pip-modules/bin:/workspace/.rvm/bin:/home/gitpod/.sdkman/candidates/maven/current/bin:/home/gitpod/.sdkman/candidates/java/current/bin:/home/gitpod/.sdkman/candidates/gradle/current/bin:/home/gitpod/.rvm/gems/ruby-2.6.6/bin:/home/gitpod/.rvm/gems/ruby-2.6.6@global/bin:/home/gitpod/.rvm/rubies/ruby-2.6.6/bin:/home/gitpod/.pyenv/plugins/pyenv-virtualenv/shims:/home/gitpod/.pyenv/shims:/workspace/.cargo/bin:/workspace/.pip-modules/bin:/workspace/.rvm/bin:/home/gitpod/.pyenv/bin:/home/gitpod/.pyenv/shims:/workspace/go/bin:/home/gitpod/go/bin:/home/gitpod/go-packages/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin/:/home/gitpod/.nvm/versions/node/v12.20.0/bin:/home/gitpod/.cargo/bin:/home/gitpod/.rvm/bin
10 verbose lifecycle @solana/web3.js@0.0.0-development~build:fixtures: CWD: /workspace/solana-web3.js
11 silly lifecycle @solana/web3.js@0.0.0-development~build:fixtures: Args: [
11 silly lifecycle   '-c',
11 silly lifecycle   'set -ex; ./test/fixtures/noop-c/build.sh; ./test/fixtures/noop-rust/build.sh'
11 silly lifecycle ]
12 silly lifecycle @solana/web3.js@0.0.0-development~build:fixtures: Returned: code: 101  signal: null
13 info lifecycle @solana/web3.js@0.0.0-development~build:fixtures: Failed to exec build:fixtures script
14 verbose stack Error: @solana/web3.js@0.0.0-development build:fixtures: `set -ex; ./test/fixtures/noop-c/build.sh; ./test/fixtures/noop-rust/build.sh`
14 verbose stack Exit status 101
14 verbose stack     at EventEmitter.<anonymous> (/workspace/solana-web3.js/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
14 verbose stack     at EventEmitter.emit (events.js:314:20)
14 verbose stack     at ChildProcess.<anonymous> (/workspace/solana-web3.js/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
14 verbose stack     at ChildProcess.emit (events.js:314:20)
14 verbose stack     at maybeClose (internal/child_process.js:1022:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
15 verbose pkgid @solana/web3.js@0.0.0-development
16 verbose cwd /workspace/solana-web3.js
17 verbose Linux 5.4.0-1024-gcp
18 verbose argv "/home/gitpod/.nvm/versions/node/v12.20.0/bin/node" "/workspace/solana-web3.js/node_modules/.bin/npm" "run" "build:fixtures"
19 verbose node v12.20.0
20 verbose npm  v6.14.10
21 error code ELIFECYCLE
22 error errno 101
23 error @solana/web3.js@0.0.0-development build:fixtures: `set -ex; ./test/fixtures/noop-c/build.sh; ./test/fixtures/noop-rust/build.sh`
23 error Exit status 101
24 error Failed at the @solana/web3.js@0.0.0-development build:fixtures script.
24 error This is probably not a problem with npm. There is likely additional logging output above.
25 verbose exit [ 101, true ]
```

tried on gitpod machine. Here's the link to it: `https://e4a3fb45-8e98-4cee-a664-7a904aaf3c0b.ws-eu03.gitpod.io/#/workspace/solana-web3.js`

**Comment by jstarry** [comment]
@aahutsal are you still hitting this issue? Perhaps you're hitting some resource constraints within gitpod? `yarn test` works fine for me locally

**Comment by aahutsal** [comment]
Yes, I do have that issue on a fresh machine, fresh Ubuntu docker box and gitpod. Guess it's related to missing rust packages or something.

**Comment by jstarry** [comment]
Should be resolved by the [v1.0.0 release](https://github.com/solana-labs/solana-web3.js/releases/tag/v1.0.0)
